java/bundles/org.eclipse.set.feature.validation/src/org/eclipse/set/feature/validation/utils/FileInfoReader.java

```diff
@@ -91,4 +107,29 @@ private Node getValueNode(final String nodeName) {
 		return null;
 	}
 
+	private List<ContainerContent> getFileInhalts() {
```

Comment:
Rename this method so it's english only (change line 72 too):
```suggestion
	private List<ContainerContent> getContainerContents() {
```